### PR TITLE
Fix Puppet strings warnings

### PR DESCRIPTION
**Pull request by treydock**
# Pull Request Checklist

## Description

Fix warnings from generating documentation with Puppet Strings.

### CODE REVIEW

**Comment by treydock** [comment]
Tests will fail until #1146 is merged.

File: REFERENCE.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ _Public Classes_
 * [`sensu`](#sensu): Base Sensu class
 * [`sensu::agent`](#sensuagent): Manage Sensu agent
 * [`sensu::backend`](#sensubackend): Manage Sensu backend
-* [`sensu::backend::datastore::postgresql`](#sensubackenddatastorepostgresql):
```
Comment (ghoneycutt) [inline REFERENCE.md]: REFERENCE.md is created during a release and should not be part of the PR.

Comment (treydock) [reply REFERENCE.md]: Fixed, had generated to verify the change and accidentally committed.
